cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowImpl2.qll

```diff
@@ -335,6 +341,29 @@ private predicate outBarrier(NodeEx node, Configuration config) {
   )
 }
 
+/** A bridge class to access the deprecated `isBarrierGuard`. */
+private class BarrierGuardGuardedNodeBridge extends Unit {
```

Comment:
Neat trick for accessing a `deprecated` predicate without getting a deprecation warning.